core/services/offchainreporting/common.go

```diff
@@ -10,7 +11,7 @@ func NewLocalConfig(cfg ValidationConfig, spec job.OffchainReportingOracleSpec)
 	lc := ocrtypes.LocalConfig{
 		BlockchainTimeout:                      spec.BlockchainTimeout.Duration(),
 		ContractConfigConfirmations:            spec.ContractConfigConfirmations,
-		SkipContractConfigConfirmations:        cfg.Layer2Type() != "",
+		SkipContractConfigConfirmations:        types.IsL2(cfg.ChainType()),
```

Comment:
I still think `chain.IsL2` reads slightly better but I'm fine with this too

Reply:
Yeah I thought so too, but I think adding another interface method required duplicating a bunch of the same check around. If we do a `type  ChainType string`, then it could be a method on that: `cfg.ChainType().IsL2()`

Reply:
Now: `cfg.ChainType().IsL2()`